includes/class-sensei-modules.php

```diff
@@ -1268,8 +1268,8 @@ public function get_course_modules($course_id = 0) {
      */
     public function enqueue_styles() {
 
-        wp_register_style($this->taxonomy . '-frontend', esc_url($this->assets_url) . 'css/modules-frontend.css', Sensei()->version );
-        wp_enqueue_style($this->taxonomy . '-frontend');
+        // wp_register_style($this->taxonomy . '-frontend', esc_url($this->assets_url) . 'css/modules-frontend.css', Sensei()->version );
```

Comment:
Yep, at the time of the PR I wasn't sure if Dwain was still aiming to do something with that, there some Slack discussions on this if I recall correctly. 

In Sensei Styles Pro I also had difficulty with hooks to remove its enqueue'ing (hence the hack'ish approach there) - The [recommended method](https://docs.woothemes.com/document/sensei-hooks-actions-and-filters/#disable-the-default-sensei-styles) isn't applicable to the modules stylesheet :|